Assets/Scripts/Framework/Train/TrainEngineSpeedTypes.cs

```diff
@@ -0,0 +1,18 @@
+namespace DerailedDeliveries.Framework.Train
+{
+    /// <summary>
+    /// Enum state for different train speeds.
+    /// </summary>
+    public enum TrainEngineSpeedTypes
+    {
+        HIGH_REVERSE,
+        MEDIUM_REVERSE,
+        LOW_REVERSE,
+        
+        STILL,
+        
+        LOW,
+        MEDIUM,
+        HIGH
```

Comment:
Change these enum constants to `UpperCamelCase`. [wiki](https://github.com/lucasvdiepen/DerailedDeliveries/wiki/Code-Conventions#enums).